<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteSet</class>
 <widget class="QMainWindow" name="remoteSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>遥控列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>199</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>主遥控</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vice3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mainID">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>238</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vice4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vice5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>478</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vice1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>副遥控</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vice2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设备ID：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bindviceButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 93, 139);</string>
    </property>
    <property name="text">
     <string>绑定副遥控</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bindmainButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 93, 139);</string>
    </property>
    <property name="text">
     <string>绑定主遥控</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearmain">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(241, 161, 0);</string>
    </property>
    <property name="text">
     <string>解绑</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_main" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>861</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <widget class="QLabel" name="label_remotelable">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(6, 93, 139);</string>
     </property>
     <property name="text">
      <string>遥控搜索中</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_mainORvice">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主遥控</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ID">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*设备ID:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mainbindCencel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(6, 93, 139);</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mainbindOK">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(227, 151, 0);</string>
     </property>
     <property name="text">
      <string>绑定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bindID">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_mainRemind">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>511</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>绑定说明：长按遥控器&quot;停机&quot;按键3秒</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_remote">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/back/Image/button/839.png);
background-color: rgb(183, 222, 242);</string>
    </property>
    <property name="text">
     <string>遥控设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_system">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/back/Image/button/842.png);
background-color: rgb(9, 82, 120);</string>
    </property>
    <property name="text">
     <string>系统信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>保利科技园A栋2单元</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_control">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(9, 82, 120);
background-image: url(:/back/Image/button/837.png);</string>
    </property>
    <property name="text">
     <string>分控操作</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_WiFi">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back/Image/button/four.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dairy">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/back/Image/button/840.png);
background-color: rgb(9, 82, 120);</string>
    </property>
    <property name="text">
     <string>操作日志</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_oneConfig">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/back/Image/button/838.png);
background-color: rgb(9, 82, 120);</string>
    </property>
    <property name="text">
     <string>分控设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_project">
    <property name="geometry">
     <rect>
      <x>614</x>
      <y>40</y>
      <width>132</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/back/Image/button/841.png);
background-color: rgb(9, 82, 120);</string>
    </property>
    <property name="text">
     <string>项目信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
